src/LanguageLine.php

```diff
@@ -15,7 +15,7 @@ class LanguageLine extends Model
     public $guarded = ['id'];
 
     /** @var array */
-    protected $casts = ['text' => 'array'];
+    // protected $casts = ['text' => 'array'];
```

Comment:
Why is this removed?

Reply:
$casts was removed because when we save the model if text column value contain language other than english it will save as unicode characters i want to save this values as it is in database
for example
LanguageLine::create(['group' => 'button','key' => 'reset','text' => ['en' => 'Reset','ar' => 'إعادة ضبط']]);

the above line save result in text column like this {"en":"Reset","ar":"\u0625\u0639\u0627\u062f\u0629 \u0636\u0628\u0637"}

for ar language its save value as unicode characters

So i did changes in code i write accessor and mutator for text column 
    public function setTextAttribute($value){
        $this->attributes['text'] = json_encode($value,JSON_UNESCAPED_UNICODE); //this code is important
    }

    public function getTextAttribute($value){
        return json_decode($value,true);
    }

Reply:
> Can you add a test for this?

Can you please explain how to write test cases

Reply:
> utf 8 encoding
> […](#)
> On Tue, May 11, 2021, 2:05 AM Adam Levenson ***@***.***> wrote: @surendra-17 <https://github.com/surendra-17> what encoding is your database? I only tried it with some German and French characters and it worked when I switched the migration to a JSON column type. — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#127 (comment)](https://github.com/spatie/laravel-translation-loader/pull/127#issuecomment-837296971)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ALPT2NAMKHGWKDAALZSJG3DTNA7P3ANCNFSM44NFWGQA> .

I'm also facing the same issue with UTF-8, sometimes a solution is to use a json column, but in MariaDB there is no option, because JSON columns are only available for MySQL.

Reply:
https://github.com/surendra-17/laravel-translation-loader use this fork i fixed the issue